examples/src/components/story/story-page.tsx

```diff
@@ -24,7 +24,6 @@ const useStyles = makeStyles({
     backgroundColor: tokens.colorNeutralBackground3,
     display: "grid",
     gridTemplateColumns: "1fr min-content",
-    ...shorthands.padding(0, "15%", 0, "5%"),
```

Comment:
needed?

Reply:
No. Reverted. But a lot of screen space is wasted IMO.